test/integration/testPutGet.js

```diff
@@ -857,3 +861,315 @@ describe('PUT GET test multiple files', function () {
     setTimeout(checkResult, 100);
   }
 });
+
+describe('PUT GET test without compress', function () {
+  let connection;
+  const stage = `@${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
+  const createTable = `create or replace table ${TEMP_TABLE_NAME} (${COL1} STRING, ${COL2} STRING, ${COL3} STRING)`;
+  const removeFile = `REMOVE ${stage}`;
+  const dropTable = `DROP TABLE IF EXISTS ${TEMP_TABLE_NAME}`;
+
+  // Create a temp file without specified file extension
+  const tmpFile = tmp.fileSync();
+  // Write row data to temp file
+  fs.writeFileSync(tmpFile.name, ROW_DATA);  
```

Comment:
I think we should run it in before section - in describe body we should only define contants/variables

Reply:
updated